api/v1/routes.go

```diff
@@ -138,6 +139,15 @@ func postLayer(w http.ResponseWriter, r *http.Request, p httprouter.Params, ctx
 }
 
 func getLayer(w http.ResponseWriter, r *http.Request, p httprouter.Params, ctx *context.RouteContext) (string, int) {
+
+	log.Info(updater.DbFinishUpdate)
+
+	if updater.DbFinishUpdate {
+		writeResponse(w, r, http.StatusNotFound, LayerEnvelope{Error: & Error{"DB is being updated so no query to it can be done"}})
```

Comment:
The message could be clearer. Something like: `The initial vulnerability update has not finished yet, therefore this API call is currently unavailable. Please try again later`. Also, the error code should probably be 503.